_sass/settings/_variables.scss

```diff
@@ -30,6 +30,7 @@ $dark-indigo: #0d2857;
 $dusty-grey: #999;
 $scorpion: #5b5b5b;
 $white: #fff;
+$light-grey: #eee;
```

Comment:
Je vais casser les pieds, mais : http://chir.ag/projects/name-that-color/#EEEEEE